updating a kas-grant panics

I just tried to update(assign) a kas-grant but it failed with the following error.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x30 pc=0x104c43614]

goroutine 1 [running]:
github.com/opentdf/otdfctl/pkg/cli.FlagHelper.GetRequiredString({0x14000bd8388?}, {0x106c84313, 0x3})
        /home/runner/go/pkg/mod/github.com/opentdf/otdfctl@v0.9.0/pkg/cli/flagValues.go:26 +0x24
github.com/opentdf/otdfctl/cmd.policy_updateKasGrant(0x14000bd8388, {0x106c85736?, 0x4?, 0x106c8566a?})
        /home/runner/go/pkg/mod/github.com/opentdf/otdfctl@v0.9.0/cmd/kas-grants.go:17 +0x25c
github.com/spf13/cobra.(*Command).execute(0x14000bd8388, {0x140002a00e0, 0x7, 0x7})
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:989 +0x828
github.com/spf13/cobra.(*Command).ExecuteC(0x10e7ee540)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1117 +0x344
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041
github.com/virtru-corp/data-security-platform/cmd.Execute()
        /home/runner/_work/data-security-platform/data-security-platform/cmd/root.go:56 +0xf0
main.main()
```

I think the issue might be around have attribute id and value id required when only one of those is necessary. 

https://github.com/opentdf/otdfctl/blob/7f7bc70c625bcb429ab1f11d917e7b37a438736e/docs/man/policy/kas-grants/update.md?plain=1#L8-L19

--- a/cmd/kas-grants.go
+++ b/cmd/kas-grants.go
@@ -12,10 +12,9 @@ func policy_updateKasGrant(cmd *cobra.Command, args []string) {
 
 	flagHelper := cli.NewFlagHelper(cmd)
 
-	attr := flagHelper.GetOptionalString("attribute")
-	val := flagHelper.GetOptionalString("value")
-	kas := flagHelper.GetRequiredString("kas")
-
+	attr := flagHelper.GetOptionalString("attribute-id")
+	val := flagHelper.GetOptionalString("value-id")
+	kas := flagHelper.GetRequiredString("kas-id")
 	if attr == "" && val == "" {
 		cli.ExitWithError("Must specify and Attribute Definition id or Value id to update.", nil)
 	}
@@ -51,9 +50,9 @@ func policy_deleteKasGrant(cmd *cobra.Command, args []string) {
 	defer h.Close()
 
 	flagHelper := cli.NewFlagHelper(cmd)
-	attr := flagHelper.GetOptionalString("attribute")
-	val := flagHelper.GetOptionalString("value")
-	kas := flagHelper.GetRequiredString("kas")
+	attr := flagHelper.GetOptionalString("attribute-id")
+	val := flagHelper.GetOptionalString("value-id")
+	kas := flagHelper.GetRequiredString("kas-id")
 
 	if attr == "" && val == "" {
 		cli.ExitWithError("Must specify and Attribute Definition id or Value id to delete.", nil)

